x-pack/plugins/apm/public/components/shared/charts/spark_plot/index.tsx

```diff
@@ -68,7 +68,16 @@ export function SparkPlot({
     <EuiFlexGroup gutterSize="m" responsive={false}>
       <EuiFlexItem grow={false}>
         {!series || series.every((point) => point.y === null) ? (
-          <EuiIcon type="visLine" color="subdued" style={chartSize} />
+          <div
+            style={{
+              ...chartSize,
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: 'center',
+            }}
+          >
+            <EuiIcon type="visLine" color={theme.eui.euiColorLightShade} />
```

Comment:
```suggestion
            <EuiIcon type="visLine" color={theme.eui.euiColorMediumShade} />
```

I mistook the variable name, we need to make this slightly darker.